Dockerfile

```diff
@@ -14,8 +14,6 @@ ENV BUNDLE_JOBS 8
 ENV APP_HOME /myapp
 
 ARG RAILS_ENV=production
-ENV RAILS_ENV=$RAILS_ENV
-ENV RACK_ENV=$RAILS_ENV
```

Comment:
`arg` is only available during build, but having those two `env`s here makes them permanent.

This can have an undesired effect in the test environment. In my testings, having those two lines make the test environment use the development database. Which is not a problem because everything runs inside a transaction but it is not ideal.

@parndt, please make sure you have both of them in fly :v:

Reply:
Actually, I take it back. I don't think my test environment was using development database because of these two lines.

Yet, I don't think it is a good idea to have the two `env`s because it makes them permanent.